backend/src/schema/resolvers/notifications.spec.js

```diff
@@ -125,9 +202,30 @@ describe('given some notifications', () => {
             ... on Comment {
               content
             }
+            ... on Report {
+              id
+              filed {
+                reasonCategory
+                reasonDescription
+                reportedResource {
+                  __typename
+                  ... on User {
+                    id
+                    name
+                  }
+                  ... on Post {
+                    id
+                    title
+                    content
+                  }
+                  ... on Comment {
+                    id
+                    content
+                  }
+                }
+              }
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T18:59:10Z" title="Thursday, January 23rd 2020, 7:59:10 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -145,80 +233,188 @@ describe('given some notifications', () => {
 
       describe('no filters', () => {
         it('returns all notifications of current user', async () => {
-          const expected = [
-            {
-              from: {
-                __typename: 'Comment',
-                content: 'You have seen this comment mentioning already',
-              },
-              read: true,
-              createdAt: '2019-08-30T15:33:48.651Z',
-            },
-            {
-              from: {
-                __typename: 'Post',
-                content: 'Already seen post mention',
-              },
-              read: true,
-              createdAt: '2019-08-30T17:33:48.651Z',
-            },
-            {
-              from: {
-                __typename: 'Comment',
-                content: 'You have been mentioned in a comment',
-              },
-              read: false,
-              createdAt: '2019-08-30T19:33:48.651Z',
-            },
-            {
-              from: {
-                __typename: 'Post',
-                content: 'You have been mentioned in a post',
-              },
-              read: false,
-              createdAt: '2019-08-31T17:33:48.651Z',
-            },
-          ]
-
-          await expect(query({ query: notificationQuery, variables })).resolves.toMatchObject({
+          const expected = {
             data: {
-              notifications: expect.arrayContaining(expected),
+              notifications: expect.arrayContaining([
```


do we need `arrayContaining` anymore if we are sorting by `createdAt`... or maybe it's by `updatedAt`... either way, maybe we could create the notifications with `updatedAt` and expect them to be returned in the correct, sorted order?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-24T12:32:48Z" title="Friday, January 24th 2020, 1:32:48 pm +01:00">Jan 24, 2020</time>_

---


It is sorted by `updatedAt`.
I tried out and sorted it the correct way. I added this property as well etc. …
But I couldn’t get it to work. I fell kind of stupid, but I will not spend hours on it.
If you find my mistake or misunderstanding, please go ahead !!! Your welcome!

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-05T10:13:46Z" title="Wednesday, February 5th 2020, 11:13:46 am +01:00">Feb 5, 2020</time>_

---


Solved with your help @mattwr18 😍